lib/streamlit/elements/heading.py

```diff
@@ -57,6 +60,7 @@ def header(
         anchor : str
             The anchor name of the header that can be accessed with #anchor
             in the URL. If omitted, it generates an anchor using the body.
+            If False, the anchor is not shown in the UI.
```

Comment:
@snehankekre new docstring line. Can you look at it?

Reply:
Docstring changes LGTM 👍